dask/dataframe/io/tests/test_json.py

```diff
@@ -84,6 +84,25 @@ def test_write_json_basic(orient):
         assert_eq(out, df)
 
 
+def test_to_json_with_get():
+    from dask.multiprocessing import get as mp_get
+
+    flag = [False]
+
+    def my_get(*args, **kwargs):
+        flag[0] = True
+        return mp_get(*args, **kwargs)
+
+    df = pd.DataFrame({"x": ["a", "b", "c", "d"], "y": [1, 2, 3, 4]})
+    ddf = dd.from_pandas(df, npartitions=2)
+
+    with tmpdir() as dn:
+        ddf.to_json(dn, compute_kwargs={"scheduler": my_get})
+        assert flag[0]
+        result = dd.read_json(os.path.join(dn, "*")).compute().reset_index(drop=True)
+        assert_eq(result, df)
```

Comment:
We try to avoid computing a dask dataframe before passing it to `assert_eq` as internally `assert_eq` will compute the dataframe and also perform some additional checks. Is this `compute()` call required, or would something like

```python
result = dd.read_json(os.path.join(dn, "*"))
assert_eq(result, df, check_index=False)
```

work instead?

Reply:
Fixed! I took the code as-is from `test_csv.py`, so I've fixed it in that test as well.

Reply:
Awesome, thanks